pyproject.toml

```diff
@@ -37,6 +37,7 @@ click = "^7.1.2"
 "ruamel.yaml" = "^0.17.21"
 python-Levenshtein = "^0.12.2"
 rudder-sdk-python = "^1.0.3"
+PyJWT = ">=2.4.0"
```

Comment:
This is a transitive dependency - we don't actually use it anywhere. Here's where it's coming from:

![image](https://user-images.githubusercontent.com/3487156/172451138-ed166ff1-a1ad-4722-8d51-bb5a7ce06dee.png)

Looks like these are test environment requirements (see https://github.com/snowflakedb/snowflake-connector-python/blob/main/tested_requirements/requirements_39.reqs#L11 for an example - the readme in there indicates these are for testing alone) - the snowflake-sqlalchemy dependency comes from the snowflake-connector so it's the same deal.

Reply:
Ok, looks like if we update line 22 to be 2.7.8 this should get resolved along the way.